section2/reflection.md

```diff
@@ -1,15 +1,17 @@
 3. In this section, we removed some of the supports that you had in Section 1. We didn't give the directions for how to run a file in node and made some directions a bit more vague, intentionally. How did that feel? What was uncomfortable about it? How did it support your learning?
-
+* It was a bit uncomfortable and I found that my detailed note taking was valuable and helped me a lot. I did have to refer back to different sections and even homework but I think overall it was good to re-read some old material.
 4. What is a conditional statement? Give one example of a daily life conditional. Give one example of where a conditional is probably used in a web application you use.
-
+* Conditional statements help JavaScript determine whether or not pieces of code can run. For a daily life I think how a conditional might work is if your car is not running smoothly first check if your car needs an oil change if that is true then just get an oil change if that is false then run a diagnostic test and determine what else could be going on. For a web applications I think conditional statements are used when to verify age often. For example if a user wanted to apply for a credit card online there will probably be a conditional statement in the code that determines if a users age if greater or equal to 18: if (age >= 18)
 5. How do you add multiple conditions to an `if` statement? In your own words, explain how to program reads them and determines what to do.
-
+* You would need the help of logical operators which are 'AND &&' and 'OR ||'. The && operator evaluates if both boolean values are true otherwise it evaluates everything else as false, the || operator evaluates if either of the booleans are true and it evaluates false if both booleans are false.
 6. What tools are available to you, as a developer, if you want to check that TWO conditions are met? Or, if you want to check that 1 of 2 conditions are met, before running a block of code?
-
-7. What questions do you still have about `if` statements and/or functions?
\ No newline at end of file
+* I think you would have to run an OR operation || if you wanted to check if 1 of 2 conditions are met. Although, I haven't tried this yet (we did have a brief intro to this in lesson 4) I know you can use Chrome Developer Tools to de-bugg codes.
+7. What questions do you still have about `if` statements and/or functions?
+* Not going to lie, the `if` statements were not a walk in the park and reading ahead on that articles Marius Schulz wrote I know that they can get a little more complex. Something that was hard to wrap my brain around was using logical operators with non-boolean values (this might be getting ahead of myself), is this something we will be getting a lesson on? I think it would be helpful not just to read it but having an instructor go over this concept.
```

Comment:
Yes, y'all will talk about "truthy" and "falsy" in Mod 1. And I get it, yes this is a LOT. I also think that once you start applying these concepts into REAL projects - little web apps - it will come together and make a lot more sense. You have the general concept and syntax, so you're in a great place to get a little more advanced and apply these things in your Mod 1 work.